zookeeper-server/src/test/java/org/apache/zookeeper/server/quorum/UnifiedServerSocketTest.java

```diff
@@ -382,7 +382,7 @@ public void testConnectWithoutSSLToNonStrictServerPartialWrite() throws Exceptio
     }
 
     /**
-     * Attempting to connect to a SSL-only server without SSL should fail.
+     * Attempting to connect to an SSL-only server without SSL should fail.
      */
```

Comment:
- It's weird that SSL doesn't start with a vowel(a,e,i,o,u), but use **an**
- @eolivelli  It's now not trivial to merge?  :D